Realm/RLMRealm.h

```diff
@@ -495,6 +496,22 @@ __deprecated_msg("Use `performMigrationForConfiguration:error:`") NS_REFINED_FOR
  */
 + (BOOL)performMigrationForConfiguration:(RLMRealmConfiguration *)configuration error:(NSError **)error;
 
+/**
+ Creates a `RLMThreadHandover` from the passed in objects that can be safely imported on a separate thread.
+
+ Used to pass objects between threads in a thread-safe manner.
+
+ @warning Note that the returned `ThreadHandover` must be imported at most once, and that the current version of the
+ Realm will remain pinned until this package is imported or deallocated.
+
+ @param objects The objects to pass between threads.
+ @return A package that can be imported on a separate thread.
+
+ @see RLMThreadHandover
+ */
+- (RLMThreadHandover *)exportThreadHandoverWithObjects:(NSArray<id<RLMThreadConfined>> *)objects
```

Comment:
I don't have any great ideas. I think it ideally should contain the word "thread" for the sake of people ctrl-fing the docs, should somehow indicate that it gives new references to the objects, should indicate that you can pass multiple types of things into it.

Some assorted bad ones that may inspire something:

  * `threadSafeCopy:`
  * `threadSafePackagedCopy:`
  * `threadSafePackage:`
  * `unThreadConfinedCopy:` (ugh...)
  * `exportThreadSafeReferences:`